code/extract_data.py

```diff
@@ -58,7 +58,7 @@ def process_paper(paper):
 
 # print(len(data)) # 19458
 
-
+# don't export all files into one large file. break it up. 
 def exportdata(data, path):
```

Comment:
It would be better if the export data function created a different json file for each paper instead of one giant research list of dictionaried in a json file.